### PR TITLE
OutputFileNotFoundException created and WriteTextFile Exceptions corrected

### PR DESCRIPTION
**Pull request by NassimBounouas**

### CODE REVIEW

**Review by sticmac — APPROVED** [review]

File: src/brainfuck/exceptions/OutputFileNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	 */
+	public OutputFileNotFoundException(FileNotFoundException e) {
+		super(e.getMessage());
+	}
+
+	/**
+	 * Returns the error code.
+	 * The application will exit with this error code.
+	 *
+	 * @return 3.
```
Comment (piernov) [inline src/brainfuck/exceptions/OutputFileNotFoundException.java]: return 9, not 3

File: src/brainfuck/io/WriteTextFile.java [hunk 2]
```diff
@@ -43,8 +44,10 @@ public void write(String str){
 			fos.write(str.getBytes());
 			fos.write('\n');
 			fos.close();
-		}catch(Exception e){
-			e.printStackTrace();
+		}catch(FileNotFoundException e){
```
Comment (piernov) [inline src/brainfuck/io/WriteTextFile.java]: Spaces before and after curly braces: `} catch(FileNotFoundException e) {`

File: src/brainfuck/exceptions/OutputFileNotFoundException.java (hunk 1 above)
Comment (NassimBounouas) [reply src/brainfuck/exceptions/OutputFileNotFoundException.java]: Yes as I suggested in IRC, It's a oversight on my part.

File: src/brainfuck/io/WriteTextFile.java (hunk 2 above)
Comment (NassimBounouas) [reply src/brainfuck/io/WriteTextFile.java]: Another oversight on my part.

**Pull request merged**